comms/xlog/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                xlog
+version             2.0.14beta4
+
+categories          comms
+license             GPL-3
+platforms           darwin
+maintainers         nomaintainer
+
+description         A logging program for Amateur Radio Operators.
+long_description    Xlog will display a log with the most recent contacts added to the top. You can add, delete and edit log entries. The log is saved into a text file. You can open, close and create logs as you need them. Xlog supports cabrillo, ADIF, trlog (format also used by tlf), and EDI (ARRL VHF/UHF contest format) and can import twlog, editest and OH1AA logbook files.
+
+homepage            http://www.nongnu.org/xlog/
+master_sites        https://download.savannah.gnu.org/releases/xlog/
+
+checksums           rmd160  a1a78a90c74d7f73ea680fe80a324eaea3e36e95 \
+                    sha256  0f22ebae56f7d0ef0cac6ae7047ec3fe6b330dc68dd7b25ed2c23fdc1c52a10f
+
+patchfiles          patch-configure.diff
+
+depends_lib         port:hamlib \
+                    port:pkgconfig \
```

Comment:
Unless this port needs `pkgconfig` at runtime, this should probably be `depends_build port:pkgconfig`.